doc/articles/Uno-UI-Performance.md

```diff
@@ -83,7 +83,7 @@ Here's what to look for:
 ## Advanced performance Tracing
 
 ### Profiling applications
-A profiling guide for Uno Platform apps is [available here](articles/guides/profiling-applications.md).
+A profiling guide for Uno Platform apps is [available here](articles/guides/profiling-applications.html).
```

Comment:
Thanks! Actually, all links should be using the .md extension but in this case, it's mostly caused by the relative path being incorrect.